enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -117,21 +126,53 @@ def list(self, request, *args, **kwargs):
         """
         return super().list(request, *args, **kwargs)
 
+    @extend_schema(
+        tags=[SUBSIDY_ACCESS_POLICY_CRUD_API_TAG],
+        summary='Soft-delete subsidy access policy by UUID.',
+        request=serializers.SubsidyAccessPolicyDeleteRequestSerializer,
+        responses={
+            status.HTTP_200_OK: serializers.SubsidyAccessPolicyResponseSerializer,
+            status.HTTP_404_NOT_FOUND: None,
+        },
+    )
+    @permission_required(SUBSIDY_ACCESS_POLICY_WRITE_PERMISSION, fn=policy_permission_detail_fn)
+    def destroy(self, request, *args, uuid=None, **kwargs):
+        """
+        Soft-delete a single `SubsidyAccessPolicy` record by uuid.
+        """
+        # Collect the "reason" query parameter from request body.
+        request_serializer = serializers.SubsidyAccessPolicyDeleteRequestSerializer(data=request.data)
+        request_serializer.is_valid(raise_exception=True)
+        delete_reason = request_serializer.data['reason']
+
+        try:
+            policy_to_soft_delete = self.get_queryset().get(uuid=uuid)
+        except SubsidyAccessPolicy.DoesNotExist:
+            return Response(None, status=status.HTTP_404_NOT_FOUND)
+
+        if policy_to_soft_delete.active:
+            policy_to_soft_delete.active = False
+            policy_to_soft_delete._change_reason = delete_reason  # pylint: disable=protected-access
+            policy_to_soft_delete.save()
```

Comment:
nit: might be nice to override the `delete()` method of the model itself to do this.  That way, it's more reusable for any other places we might need to delete policies from.